packages/react-native-gesture-handler/src/web/tools/NodeManager.ts

```diff
@@ -35,9 +35,8 @@ export default abstract class NodeManager {
     if (!(handlerTag in this.gestures)) {
       return;
     }
-
+    GestureHandlerOrchestrator.instance.deleteGesture(handlerTag);
```

Comment:
I don't think it is the best place to call this, wouldn't it work if we do this in `onDestroy` method?